.husky/pre-commit

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env sh
```

Comment:
lol ended up removing this after finding it's not as helpful?

Reply:
haha i did but then i thought i stashed it, seems to have made it in. Woops